solidity/deploy/01_deploy_tbtc_v2_token.ts

```diff
@@ -0,0 +1,21 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // This deployment is named TBTCTokenV2 to avoid conflict with TBTCToken
+  // deployment which represents the tBTC v1 token shipped via an external
+  // dependency.
+  await deploy("TBTCTokenV2", {
```

Comment:
@pdyraga is this fine if we name the artifact `TBTCTokenV2.json` to distinguish it from V1's `TBTCToken`?

Reply:
I think just `tBTC.json` and we will have to rename the token contract to `tBTC` as well.

Reply:
We also have the same problem with `VendingMachine`.

Reply:
I would have just `VendingMachine`. We are operating in the context of a new tBTC package version and there is only one `VendingMachine` here.

Reply:
https://github.com/keep-network/tbtc-v2/pull/24

Reply:
https://github.com/keep-network/tbtc-v2/pull/17/commits/df5ddf3d6adeffd778b7358c94b9f157a89e53c5